senza/configuration.py

```diff
@@ -0,0 +1,76 @@
+from collections.abc import MutableMapping
+from pathlib import Path
+from typing import Dict, Tuple
+
+import yaml
+from click import get_app_dir
+
+from .exceptions import InvalidConfigKey
+
+
+class Configuration(MutableMapping):
```

Comment:
I wanted to support sections because I wanted both to stay consistent with git and allow extensions\templates to have their own configuration. The stups_cli config is also dangerous for config commands since it loads configuration from the environment (which is a nice feature) but then dumps the whole configuration to the disk when saving, which would mean that users could accidentally overwrite their configuration with values that they intended to use only temporarily.